csp-mixin/panels/gcpvpc.libsonnet

```diff
@@ -0,0 +1,134 @@
+local g = import '../g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+{
+  new(this): {
+    _serviceTableCommon(valueColNiceName)::
+      g.panel.table.queryOptions.withTransformations([
+        g.panel.table.queryOptions.transformation.withId('filterFieldsByName')
+        + g.panel.table.queryOptions.transformation.withOptions({
+          include: {
+            pattern: 'service_name.*|Value.*',
+          },
+        }),
+        g.panel.table.queryOptions.transformation.withId('joinByField')
+        + g.panel.table.queryOptions.transformation.withOptions({
+          byField: 'service_name',
+          mode: 'outer',
+        }),
+      ])
+      + g.panel.table.standardOptions.withOverrides([
+        {
+          matcher: {
+            id: 'byName',
+            options: 'service_name',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Service name',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: valueColNiceName,
+            },
+          ],
+        },
+      ]),
+
+
+    gcpvpc_services_in_use_count:
+      this.signals.gcpvpc.gcpvpc_services_in_use_count.asStat()
+      + g.panel.stat.standardOptions.color.withMode('fixed')
+      + g.panel.stat.standardOptions.color.withFixedColor('text'),
+
+    gcpvpc_subnets_in_use_count:
+      this.signals.gcpvpc.gcpvpc_subnets_in_use_count.asStat()
+      + g.panel.stat.standardOptions.color.withMode('fixed')
+      + g.panel.stat.standardOptions.color.withFixedColor('text'),
+
+    gcpvpc_fixed_usage_tier_network_egress:
+      this.signals.gcpvpc.gcpvpc_fixed_usage_tier_network_egress.asTimeSeries()
+      + g.panel.timeSeries.standardOptions.withNoValue('0'),
+
+    gcpvpc_service_topk_throughput:
+      this.signals.gcpvpc.gcpvpc_service_topk_throughput_bytes.common
+      + g.panel.table.standardOptions.withUnit('bps')
+      + commonlib.panels.generic.table.base.new(
+        'Top 5 services - Average throughput',
+        [
+          this.signals.gcpvpc.gcpvpc_service_topk_throughput_bytes.asTarget()
+          + g.query.prometheus.withFormat('table')
+          + g.query.prometheus.withInstant(true),
+        ],
+        'Average throughput',
+      ) + self._serviceTableCommon('Average throughput'),
```

Comment:
I wonder, if in a future we includes a new parameter **instant** on the common-lib / **asTable** function , maybe we can stop doing this. Must of the time we use this kind of definition because we cannot define **instant** param into **asTable**. @v-zhuravlev what do you think? Should I create a new issue for this?